addons/l10n_fr_account/wizard/account_fr_fec_export_wizard.py

```diff
@@ -353,3 +356,8 @@ def generate_fec(self):
     def create_fec_report_action(self):
         # HOOK
         return
+
+    def _exclude_zero_debit_credit(self, data):
```

Comment:
Why don't you just include this in `_get_base_domain`?

Reply:
I explained here why I didn't included it in the query.
https://github.com/odoo/odoo/pull/162631#discussion_r1577804763, Please tell me if you need me to clarify something.

Reply:
Just exclude the lines that don't have `('balance', '!=', 0)`, I don't understand the issue

Reply:
`balance = debit - credit`
And `debit * credit = 0`

Reply:
Now it makes sense, Thanks!!